### DISCUSSION TITLE
Download Ver 1.2.4 but GUI is still showing 1.2.2

### DISCUSSION THREAD

**Discussion by LeG-man** [Chia-Network/chia-blockchain#8246] [Support]
Download Ver 1.2.4 but GUI is still showing 1.2.2.

**Comment by loppefaaret** [comment]
make sure you `deactivate` out of the old (venv), and `. ./activate` into the new (venv) - and update to 1.2.5 (a problem in 1.2.4 could prevent a clean GUI install)

**Comment by BrandtH22** [comment]
@LeG-man , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
